test/utils/create-droppable.js

```diff
@@ -0,0 +1,69 @@
+import { getDraggableDimension, getDroppableDimension } from '../../src/state/dimension';
+import type {
+  DroppableId,
+  DraggableDimension,
+  DraggableDimensionMap,
+  DroppableDimension,
+} from '../../src/types';
+import getClientRect from './get-client-rect';
+
+type Rect = {|
+  top: number,
+  left: number,
+  bottom: number,
+  right: number,
+|};
+
+type CreateDroppableArgs = {|
+  direction?: 'vertical' | 'horizontal',
+  droppableId: DroppableId,
+  droppableRect: Rect,
+  draggableRects: Rect[],
+|};
+
+type TestDroppable = {
+  droppableId: string,
+  droppable: DroppableDimension,
+  draggables: DraggableDimensionMap,
+  draggableIds: string[],
+  draggableDimensions: DraggableDimension[],
+};
+
+export default ({
```

Comment:
ended up moving this out as a util because I find it super handy

Reply:
this is great!